File: youmai-manager-web/src/main/webapp/js/service/specificationService.js
//规范
app.service("specificationService", function ($http) {
    this.findAll=function (page,size) {
        return $http.get('../specification/findAll.do?page=' + page + '&size=' + size)
    };

    this.update=function (entity) {
        return $http.post('../specification/update.do',entity);
    };

    this.add=function (entity) {
        return $http.post('../specification/add.do',entity);
    };

    this.findOne=function (id) {
        return $http.get('../specification/findOne.do?id=' + id)
    };

    this.deleteSpec =function (ids) {
        return $http.get('../specification/delete.do?ids='+ids);
    };

    this.selectOptionList =function () {
        return $http.get('../specification/selectOptionList.do');
    }
});

File: youmai-sellergoods-interface/src/main/java/com/youmai/sellergoods/service/SpecificationService.java
package com.youmai.sellergoods.service;


import com.youmai.pojogroup.Specification;
import entity.PageResult;

import java.util.List;
import java.util.Map;


/**
 * @Description 规格接口
 * @Date 16:21 2018/9/13
 * @Param  * @param null
 * @Author xixi
 * @return
 **/
public interface SpecificationService {

    PageResult findAll(Integer page,Integer size);

    void add(Specification specification);

    Specification findOne(Long id);

    void update(Specification specification);

    void delete(Long[] ids);

    List<Map> selectOptionList();
}
